RetrospectiveExtension.Frontend/tsconfig.json

```diff
@@ -13,7 +13,9 @@
     "noImplicitReturns": true,
     "noImplicitThis": true,
     "noImplicitAny": true,
-    "strictNullChecks": false
+    "strictNullChecks": false,
```

Comment:
Do we need to turn this one off? It doesn't seem to cause any compilation or linter issues for me if I remove it.

Reply:
That is strange. Can you try it again. I did not disable `strictNullChecks` in this PR. it was always disabled from the beginning of this project.
if you meant `esModuleInterop` then you are right it is not needed for this PR, but it would have been needed if other legacy features were used so I removed it.

Reply:
My bad, you're right. Closing.